plugins/module_utils/utils.py

```diff
@@ -23,9 +23,10 @@
     recursive_diff,
 )
 
+# pylint: disable=unused-import
 from ansible.module_utils.common.dict_transformations import (
     snake_dict_to_camel_dict,
-)  # pylint: disable=unused-import
```

Comment:
It should work this way with two spaces before adding this `# pylint: disable=unused-import`

Reply:
It does not for some Python versions. Hence changed the format.

Reply:
The failures for some of the py versions are seen here - https://github.com/ansible-collections/amazon.cloud/actions/runs/4801456605